.github/workflows/check-docs-build.yml

```diff
@@ -28,35 +28,28 @@ jobs:
             - '.github/**'
 
   check:
-    name: check manual
+    name: check docs build
     needs: changes
     if: github.event_name == 'schedule' || needs.changes.outputs.manual == 'true'
     runs-on: ubuntu-22.04
```

Comment:
Do you know if there is a reason why we are not using `ubuntu-latest` ?

Reply:
I'm not sure. I can use the latest if you'd prefer?